src/modules/esl-share/test/config/reference-buttons.ts

```diff
@@ -0,0 +1,164 @@
+/* eslint-disable max-len */
+export const referenceButtons = {
```

Comment:
Do we have any benefit from a plain copy of the inner config?
Not sure this gives any additional advantage for checking buttons config. 

Test for import, config size, or consistency with the object inside the config file or general type shate - that's a valid test for sure.
 But if we have a problem with a URL pattern or icon inside the button config, I believe right now we just have it twice: in test data and in the real data :)

Reply:
The benefit was when I transferred the configuration from JSON to typescript files. I managed to make 5 mistakes in values on 20 files (don't ask how! :smiling_face_with_tear:  ). These tests helped to identify errors in the values. But, when updating the existing buttons or adding new buttons, they do not make sense, I agree with you on this. Therefore, I deleted them.